xbmc/pvr/dialogs/GUIDialogPVRChannelManager.cpp

```diff
@@ -417,125 +414,6 @@ bool CGUIDialogPVRChannelManager::OnClickButtonGroupManager(CGUIMessage &message
   return true;
 }
 
-bool CGUIDialogPVRChannelManager::OnClickButtonEditChannel(CGUIMessage &message)
-{
```

Comment:
@Jalle19, the goal of this functions was primary to open a add-on channel dialogue. Due to a problem with add-on dialogue (always shown in the back) this parts become never finished from me. 

On the RDS pull request is it added in a not so good way (https://github.com/AlwinEsch/kodi/commit/175ab2e0f730f8b6d1bc4e478744d68a0f5b9fc5).

Is true that the usage of the virtual channels is not needed, but for me this buttons are good to allow edit of channels on the backend.

@opdenkamp, what think you, allow opening of addon channel dialogues over it?